aeron-driver/src/main/c/media/aeron_send_channel_endpoint.c

```diff
@@ -449,8 +456,11 @@ void aeron_send_channel_endpoint_on_nak(
 
     if (NULL != publication)
     {
-        aeron_network_publication_on_nak(publication, nak_header->term_id, nak_header->term_offset, nak_header->length);
+        return aeron_network_publication_on_nak(publication, nak_header->term_id, nak_header->term_offset, nak_header->length);
```

Comment:
Would be useful to have an `AERON_APPEND_ERR` if aeron_network_publication_on_nak returns -1.